pkg/maps/lbmap/types.go

```diff
@@ -25,6 +25,16 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// LBMap is the interface describing methods for manipulating service maps.
+type LBMap interface {
```

Comment:
Shouldn't this interface be defined where it's used? Ie in `pkg/service`? Or are there dependency problems with `pkg/loadbalancer`?

Reply:
I wasn't aware of the following recommendation:

> Go interfaces generally belong in the package that uses values of the interface type, not the package that implements those values. The implementing package should return concrete (usually pointer or struct) types: that way, new methods can be added to implementations without requiring extensive refactoring.

https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Moved it to the `pkg/service`, as we currently have just one consumer of the interface.